ProcessLib/LiquidFlow/LiquidFlowProcess.cpp

```diff
@@ -66,35 +66,20 @@ void LiquidFlowProcess::initializeConcreteProcess(
         pv.getShapeFunctionOrder(), _local_assemblers,
         mesh.isAxiallySymmetric(), integration_order, _gravitational_axis_id,
         _gravitational_acceleration, _reference_temperature,
-        *_material_properties);
+        *_material_properties, this->_coupling_term);
```

Comment:
Not sure, but is `this` required in this context?

Reply:
Removed.